ipn/ipnlocal/state_test.go

```diff
@@ -291,6 +292,47 @@ func (cc *mockControl) UpdateEndpoints(endpoints []tailcfg.Endpoint) {
 // network delays) are just ignored for now, which makes the test
 // predictable, but maybe a bit less thorough. This is more of an overall
 // state machine test than a test of the wgengine+magicsock integration.
+func TestStatusWithoutPeers(t *testing.T) {
```

Comment:
I added the test here to make use of the mock control server helper methods - but we may want to refactor and put these methods in a helper library.

Reply:
You don't need to be in the same file for the helpers to be accessible, so you can put the test in local_test.go and still use these.